src/main/java/com/musala/sdcs/devices/base/Device.java

```diff
@@ -0,0 +1,72 @@
+package com.musala.sdcs.devices.base;
```

Comment:
In general, it would be good to create a subpackages for all builders (com.musala.sdcs.devices.builder),
 commands (com.musala.sdcs.devices.command) and channels.